utilities/cloudharness_utilities/deployment-configuration/value-template.yaml

```diff
@@ -24,6 +24,8 @@ harness:
     auto: true
     name:
     port: 80
+  use_services:
+    # e.g. - name: mnp-checkout
```

Comment:
The empty value is invalid in yaml. Also better remove mnp reference. 

I can suggest for the empty array: 

`use_services: []`